### ISSUE TITLE
Predicting with base_margin

### ISSUE THREAD

**Issue by nashjojo** [dmlc/xgboost#98]
Hi Tianqi,

I am using train.base_margin to train my model, and I have a test.base_margin in the same folder. When I use the model to predict my test set, it seems all pred.txt is 0. 
How to use test.base_margin to do prediction?

Thank you.
